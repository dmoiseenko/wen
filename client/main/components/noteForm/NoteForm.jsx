import React from 'react';
import { Field } from 'redux-form';

import Input from '../../../common/components/input/Input';
import { textField } from '../../core/note.form.constant';
import NewNoteContainer from './NewNote.container';


export default function NoteForm() {
  return (
    <form className="container">
      <Field
        name={textField}
        component={Input}
        placeholder="note text"
        type="text"
        iconClassName="fa fa-pencil"
      />
      <div className="field">
        <NewNoteContainer />
      </div>
    </form>
  );
}
