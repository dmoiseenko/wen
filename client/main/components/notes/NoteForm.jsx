import React from 'react';
import { Field } from 'redux-form';

import Input from '../../../common/components/input/Input';
import { textField } from '../../core/note.form.constant';
import NewNoteContainer from '../notes/NewNote.container';


export default function NoteForm() {
  return (
    <form className="box">
      <Field
        name={textField}
        component={Input}
        placeholder="note text"
        label="new note"
        type="text"
      />
      <div className="field">
        <NewNoteContainer />
      </div>
    </form>
  );
}
