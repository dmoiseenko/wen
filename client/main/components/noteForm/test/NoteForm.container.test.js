import * as container from '../NoteForm.container';


it('should have proper form', () => {
  expect(container.form).toMatchSnapshot();
});
