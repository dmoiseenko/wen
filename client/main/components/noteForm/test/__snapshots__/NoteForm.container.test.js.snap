// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have proper form 1`] = `
Object {
  "form": "noteForm",
  "initialValues": Object {
    "textField": "",
  },
}
`;
